build.gradle

```diff
@@ -81,6 +87,12 @@ dependencies {
     compile 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
     compile 'com.fasterxml.jackson.core:jackson-annotations:2.10.0'
 
+    compile 'com.github.manevolent:ts3j:master-SNAPSHOT'
```

Comment:
Although I did the exact same thing with Confort a while back, I do not like the idea of having a SNAPSHOT build being part of a stable release.  
I hope that @Manevolent will have a stable release that is fit for our purposes when `1.2.0` comes around.